app/assets/stylesheets/application.scss

```diff
@@ -13,6 +13,14 @@ strong {
   white-space: nowrap;
 }
 
+.no-padding-top {
+  padding-top: 0;
+}
+
```

Comment:
These are placeholders and will be replaced in [SASS-5949](https://jira.tools.tax.service.gov.uk/browse/SASS-5949)